sdk/tests/conformance/textures/misc/00_test_list.txt

```diff
@@ -1,5 +1,5 @@
 --min-version 1.0.4 canvas-teximage-after-multiple-drawimages.html
---max-version 1.9.9 compressed-tex-image.html
+compressed-tex-image.html
```

Comment:
Good catch on this test. It looks like the WebGL 2.0 version of the test has been deleted since it was originally integrated.